### DISCUSSION TITLE
Replacement for `wrapper` prop

### DISCUSSION THREAD

**Discussion by andrewfritz86** [gilbarbara/react-inlinesvg#175] [Q&A] [answered]
Hi all - I inherited a project that uses the `wrapper` prop with this package fairly extensively. I want to bump the package, but it looks like support for that prop was dropped here:
https://github.com/gilbarbara/react-inlinesvg/commit/5a5d38b75249efb0f11d1fe8c4e2e12ded609417

I'm not sure if there is a difference strategy for wrapping components outside of just literally wrapping them, but thought I'd check as to avoid a larger refactor.

Project is on `"react-inlinesvg": "^0.8.2"`

Thanks!

**Comment by gilbarbara** [comment, marked as answer]
Hello,

The package now returns the SVG directly, so you could use the previous wrapper as a JSX.Element wrapping the SVG.

**Reply by andrewfritz86** [reply]
Sounds good. Thanks!
